utils/rpms/daos.spec

```diff
@@ -156,11 +156,11 @@ Requires: ndctl
 # needed to set PMem configuration goals in BIOS through control-plane
 %if (0%{?suse_version} >= 1500)
 Requires: ipmctl >= 03.00.00.0423
-Requires: libpmemobj1 >= 2.1.0-1.suse1500
+Requires: libpmemobj1 >= 2.1.0-3.suse1500
 Requires: libfabric1 >= %{libfabric_version}
 %else
 Requires: ipmctl >= 03.00.00.0468
-Requires: libpmemobj >= 2.1.0-1%{?dist}
+Requires: libpmemobj >= 2.1.0-6%{?dist}
```

Comment:
Would it be better to keep the same version in both distros?
The last stable PMDK version is 2.1.0-3 as it is defined in https://github.com/daos-stack/pmdk-pkg/blob/master/pmdk.spec